src/cpp/shared/scripting/scripting.cpp

```diff
@@ -796,6 +796,18 @@ int l_getTownIDFromPos(lua_State *L) {
   return 1;
 }
 
+int l_getguildspell(lua_State *L) {	
+	town* twn = (town*)GetPointerFromLuaClassTable(L, StackIndexOfArg(1, 3));
+	int l = (int)luaL_checknumber(L, 2);
+	int n = (int)luaL_checknumber(L, 3);
+	if (twn->mageGuildSpells[l][n] != NULL)
```

Comment:
You realize mageGuildSpells[l][n] is not a pointer, right? This NULL check is nonsensical.